CIP-0005/README.md

```diff
@@ -99,12 +109,15 @@ We define the following set of common prefixes with their corresponding semantic
 
 ### Miscellaneous
 
-| Prefix       | Meaning               | Contents                                                      |
-| ---          | ---                   | ---                                                           |
-| `addr`       | Mainnet address       | Network tag, payment credential and optional stake credential |
-| `addr_test`  | Testnet address       | Network tag, payment credential and optional stake credential |
-| `stake`      | Mainnet stake address | Network tag and stake credential                              |
-| `stake_test` | Testnet stake address | Network tag and stake credential                              |
+| Prefix       | Meaning                                  | Contents                                                      |
```

Comment:
Following is the structure for drep and committee credential in ledger
```hs
data Credential (kr :: KeyRole) c
  = ScriptHashObj !(ScriptHash c)
  | KeyHashObj !(KeyHash kr c)
  deriving (Show, Eq, Generic, NFData, Ord)
  
data ConwayGovCert c
  = ConwayRegDRep !(Credential 'DRepRole c) !Coin !(StrictMaybe (Anchor c))
  | ConwayUnRegDRep !(Credential 'DRepRole c) !Coin
  | ConwayUpdateDRep !(Credential 'DRepRole c) !(StrictMaybe (Anchor c))
  | ConwayAuthCommitteeHotKey !(Credential 'ColdCommitteeRole c) !(Credential 'HotCommitteeRole c)
  | ConwayResignCommitteeColdKey !(Credential 'ColdCommitteeRole c) !(StrictMaybe (Anchor c))
```

Since it is necessary to provide the type `Credential` when creating `GovCert` object, We should include the network tag header in `drep`, `cc_hot` and `cc_cold` encoding like in `stake`. 

**Note :** Even though the header byte is called `Network tag`, it also information about the type of credential (whether its script hash or key hash) in one of the bits.

Reply:
Good point!

I agree, I believe we should have to introduce a header tag for DRep credentials, CC hot and CC cold creds.

The question is, do we also add a bit to differentiate between networks?

Reply:
Only reward accounts have Network Tag. Credential never had this tag. 
Why do you think network tag should be there?

Reply:
oh - I am confused.

I was thinking more along the lines of the address headers, specifically [
Shelley type 6 and 7](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0019#shelley-addresses), these do not have stake credentials yet they have network tags.

Reply:
ahh the protocol already includes differentiation between credential types